specification/sbi_atee.adoc

```diff
@@ -13,6 +43,858 @@ running a VMM).
 The following operations must be implemented by the TSM ( _Formal input 
 parameters and return codes are TBD_ ).
 
+== TEE Extension (EID #0x41544545)
+
+=== Function: Create TVM (FID #0)
+[source, C]
+-----
+struct sbiret sbi_tee_create_tvm(unsigned long tvm_create_params,
+                                 unsigned long tvm_create_params_len);
+-----
+Creates a confidential TVM using the specified parameters. The tvm_create_parameters
+is a pointer to the `tvm_create_params` described below, and tvm_create_params_len is
+the size of the structure in bytes. 
+
+Callers of this API should first invoke `sbi_tee_tsm_get_info()` to obtain information
+about the parameters that should be used to populate `tvm_create_params`.
+
+[source, C]
+----
+struct tvm_create_params {
+    /* 
+    The base physical address of the 16KiB confidential memory region
+    that should be used for the TVM’s page directory. Must be 16KiB-aligned.
+    */
+    unsigned long tvm_page_directory_addr;
+    /* 
+    The base physical address of the confidential memory region to be used
+    to hold the TVM’s global state. Must be page-aligned and the number of
```

Comment:
What is the global state of the TVM ? It is not explained anywhere.

Reply:
I have removed global, and added some exposition...PTAL.